internal/testplanet/planet.go

```diff
@@ -79,6 +79,11 @@ type Planet struct {
 	cancel func()
 }
 
+// SatelliteSystem contains all the processes needed to run a full Satellite setup
+type SatelliteSystem struct {
```

Comment:
I just realized that this could be just named `testplanet.Satellite`, might end up looking nicer in code needing to use it. But `SatelliteSystem` is fine as well.